authors/settings.py

```diff
@@ -147,5 +153,55 @@
     ),
 }
 
+AUTHENTICATION_BACKENDS = [
+    'rest_framework_social_oauth2.backends.DjangoOAuth2',
+   'django.contrib.auth.backends.ModelBackend',
+]
+
+
+
+SOCIAL_AUTH_GOOGLE_OAUTH2_KEY = os.environ['SOCIAL_AUTH_GOOGLE_OAUTH2_KEY']
+SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET = os.environ['SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET']
+#Scope
+SOCIAL_AUTH_GOOGLE_OAUTH_SCOPE = ['email', 'username']
+
+SOCIAL_AUTH_TWITTER_KEY = os.environ['SOCIAL_AUTH_TWITTER_KEY']
```

Comment:
Add these env configs to the env.example file in case someone wants to run the stuff locally

Reply:
@JoanNgatia added as advised.